app/routes/playlists.js

```diff
@@ -0,0 +1,30 @@
+import SubRoute from 'backbone.subroute';
+import application from '../application';
+
+let PlaylistsRouter = SubRoute.extend({
+    routes: {
+        ':slug': 'show',
```

Comment:
I think we have think of a pattern like:
- `:slug/tracks`: display the playlist tracks
- `:slug/edit`: edit the playlist
- `:slug/delete`: remove the playlist confirm view

So this one is probably more `:slug/tracks`, nope?